src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier;
+    private final int maximumNumber = 100;
+    private final int minimumNumber = 1;
+
+    public Lottery() {
+        colorSupplier = new ColorSupplier();
```

Comment:
You random is created immediately, and colorSupplier in the constructor. Use one style of creation, or two variables at once, or two in the constructor

Reply:
not fixed